test/e2e/create_test.go

```diff
@@ -42,8 +42,8 @@ const (
 func creationTestSuite(t *testing.T) {
 	tc, err := NewTestContext()
 	require.NoError(t, err)
-	// The order of tests here are important. Any node object related tests should be run only after
-	// testWindowsNodeCreation as that initializes the node objects in the global context.
+
+	require.NoError(t, tc.loadExistingNodes(), "error getting the current Windows nodes in the cluster")
 	if !t.Run("Creation", tc.testWindowsNodeCreation) {
```

Comment:
Should we separate this into its own test suite and make this a validationTestSuite()?